src/extension/geofence/geofence-server/src/test/java/org/geoserver/geofence/server/rest/RulesRestControllerTest.java

```diff
@@ -327,7 +327,7 @@ public void testMovingRules() {
         // get the rules so we can access their id
         JaxbRuleList originalRules = controller.get(
                 0, 6, false, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null,
-                null, null);
+                null, null, null, null);
```

Comment:
This makes the test compile, but it's not really checking that the new fields can be read and written.

Reply:
Tests added